Week2/homework/maartjes_work.js

```diff
@@ -43,5 +43,70 @@ const tuesday = [
 ];
 
 const tasks = monday.concat(tuesday);
+/*
+function duration(item) {
+  return item.duration;
+}
+function sum(prev, next) {
+  return prev + next;
+}
+console.log(tasks.map(duration).reduce(sum));
+*/
+
+/*
+let hours = 0;
+for (let i = 0; i < tasks.length; i++) {
+  const num = tasks[i].duration;
+  hours += num;
+}
+hours /= 60;
+console.log(hours);
+*/
+
+let hours = 0;
+const duration = (tasks.map(tasks => tasks.duration));
+for (let i = 0; i < duration.length; i++) {
+  hours += duration[i];
+}
+hours /= 60;
+console.log(hours);
+
+//------------------
+
+
+/*
+const twoOrMoreHours = [];
+for (let i = 0; i < duration.length; i++) {
+  if (duration[i] >= 120) twoOrMoreHours.push(duration[i]);
+}
+console.log(twoOrMoreHours);
+*/
+
+
+const twoOrMoreHours = (duration.filter(values => {
+  return values >= 120;
+}
+));
+
+console.log(twoOrMoreHours);
+
+//------------------
+/*
+const salaryPerHour = '24.35';
+let salary = 0;
+
+for (let i = 0; i < duration.length; i++) {
+  salary = hours * salaryPerHour;
+}
+console.log('€ ' + salary.toFixed(2));
+
+*/
+
+const HoursArray = [hours];
```

Comment:
Instead of this array with a single element, you should use the `twoOrMoreHours` array of durations that you created earlier.